cachito/common/packages_data.py

```diff
@@ -136,6 +136,21 @@ def write_to_file(self, file_name: Union[str, Path]) -> None:
         with open(file_name, "w", encoding="utf-8") as f:
             json.dump({"packages": self._packages}, f)
 
+    def _clear_nfs_cache(self, file_name: Union[str, Path]) -> None:
+        """
+        Force the NFS to clear its cache by listing the directory's contents.
+
+        This solution is a workaround found to the issue of the file occasionally not
+        appearing to the pod that tried to read it, which only happened while using a NFS volume.
+
+        :param file_name: an absolute or relative filename that contains the packages file.
+        """
```

Comment:
Some of this (NFS, pod) is specific to the internal deployment. It is valuable info though, probably fine to keep it here

Reply:
I will make it more clear. I just thought it would be good to have an explation of why this cache clearing is needed, and this is interal deployment related, so I guess there's no avoiding it.

Reply:
+1, thanks for updating it